Core/scripts/dirac-install.py

```diff
@@ -1021,296 +1053,313 @@ def fixBuildPaths():
 """
 
   # Locate build path (from header of pydoc)
-  binaryPath = os.path.join( cliParams.targetPath, cliParams.platform )
-  pydocPath = os.path.join( binaryPath, 'bin', 'pydoc' )
+  binaryPath = os.path.join(cliParams.targetPath, cliParams.platform)
+  pydocPath = os.path.join(binaryPath, 'bin', 'pydoc')
   try:
-    fd = open( pydocPath )
+    fd = open(pydocPath)
     line = fd.readline()
     fd.close()
-    buildPath = line[2:line.find( cliParams.platform ) - 1]
-    replaceCmd = "grep -rIl '%s' %s | xargs sed -i'.org' 's:%s:%s:g'" % ( buildPath,
-                                                                          binaryPath,
-                                                                          buildPath,
-                                                                          cliParams.targetPath )
-    os.system( replaceCmd )
+    buildPath = line[2:line.find(cliParams.platform) - 1]
+    replaceCmd = "grep -rIl '%s' %s | xargs sed -i'.org' 's:%s:%s:g'" % (buildPath,
+                                                                         binaryPath,
+                                                                         buildPath,
+                                                                         cliParams.targetPath)
+    os.system(replaceCmd)
 
   except:
     pass
 
+
 def fixPythonShebang():
   """
   Some scripts (like the gfal2 scripts) come with a shebang pointing to the system python.
   We replace it with the environment one
  """
 
-  binaryPath = os.path.join( cliParams.targetPath, cliParams.platform )
+  binaryPath = os.path.join(cliParams.targetPath, cliParams.platform)
   try:
-    replaceCmd = "grep -rIl '#!/usr/bin/python' %s/bin | xargs sed -i'.org' 's:#!/usr/bin/python:#!/usr/bin/env python:g'" %  binaryPath
-    os.system( replaceCmd )
+    replaceCmd = "grep -rIl '#!/usr/bin/python' %s/bin |\
+     xargs sed -i'.org' 's:#!/usr/bin/python:#!/usr/bin/env python:g'" % binaryPath
+    os.system(replaceCmd)
   except:
     pass
 
 
-
-
 def runExternalsPostInstall():
   """
    If there are any postInstall in externals, run them
   """
-  postInstallPath = os.path.join( cliParams.targetPath, cliParams.platform, "postInstall" )
-  if not os.path.isdir( postInstallPath ):
-    logDEBUG( "There's no %s directory. Skipping postInstall step" % postInstallPath )
+  if cliParams.diracOS or cliParams.diracOSVersion:
+    postInstallPath = os.path.join(cliParams.targetPath, "postInstall")
```

Comment:
What is `targetPath` in fact ?

Reply:
where dirac is installed for example: /opt/dirac